app/javascript/src/components/Dashboard/Notes/constants.js

```diff
@@ -4,13 +4,29 @@ import * as yup from "yup";
 export const NOTES_FORM_INITIAL_FORM_VALUES = {
   title: "",
   description: "",
+  assigned_contact: null,
+  tag: null,
 };
 
 export const NOTES_FORM_VALIDATION_SCHEMA = yup.object().shape({
   title: yup.string().required("Title is required"),
   description: yup.string().required("Description is required"),
-  assigned_contact: yup.string().required("Assigned Contact is required"),
-  tag: yup.string().required("Tag is required"),
+  assigned_contact: yup
+    .object()
+    .nullable()
+    .shape({
+      label: yup.string(),
+      value: yup.string(),
+    })
+    .required("Assigned Contact is required"),
+  tag: yup
+    .object()
+    .nullable()
+    .shape({
+      label: yup.string(),
+      value: yup.string(),
+    })
+    .required("Tag is required"),
```

Comment:
Why exactly is that in your business logic you are not allowing a `note` to have multiple `tags`?

Example:
- https://handbook.neetokb.com/articles/git-and-github-essentials
- ^ This note has both the `git` and `github` tags.

Reply:
This is resolved.
I have set added isMulti prop to select. Also, I have updated validation for multiple tags using an array of objects in yup validation schema.